### ISSUE TITLE
normalize missing values

### ISSUE THREAD

**Issue by cmungall** [INCATools/biosample-analysis#26]
e.g. Not applicable, missing data, etc

**Comment by wdduncan** [comment]
Do we to replace with NaNs?

**Comment by cmungall** [comment]
No, this is pandas-specific. Let's start be looking at GSC guidelines:

https://gensc.org/mixs/

```
If a value is missing, please consider using the INSDC missing value vocabulary
Not applicable		information is inappropriate to report, can indicate that the standard itself fails to model or represent the information appropriately
Missing	Not collected	information of an expected format was not given because it has not been collected
Not provided	information of an expected format was not given, a value may be given at the later stage
Restricted access	information exists but can not be released openly because of privacy concerns
```

**Comment by turbomam** [comment]
Missing value normalization is part of EMA's harmonization pipeline:

- Null/empty handling is standardized in the `biosamples_flattened` and `biosamples_attributes` collections
- `attribute_harmonized_pairings` maps user-submitted variants (including "not applicable", "missing", "N/A") to canonical forms

**Comment by turbomam** [comment]
Correction to my earlier close comment: I overstated EMA's coverage here.

**What EMA actually does:** INSDC missing-value pattern matching (`insdc_missing_match`) exists in `new_env_triad_values_splitter.py` — but this is specific to env triad field processing, not a general missing-value normalization pass over all attributes.

**General missing value normalization** (e.g. standardizing "not applicable", "N/A", "none", "missing" across all biosample attributes) is not implemented as a systematic pipeline step in EMA that I can identify. The issue remains valid as a general-purpose gap.
